sponsorship/src/main/java/com/kyriba/conference/sponsorship/dao/PlanRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.kyriba.conference.sponsorship.dao;
+
+import com.kyriba.conference.sponsorship.domain.Plan;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+
+/**
+ * @author Aliaksandr Samal
+ */
+@Repository

```

Comment:
This annotation is useless at all in this context.

Reply:
One note: without this annotation Idea tells that there is no implementation for the interface. How to resolve this?

Reply:
It is OK, just ignore this warning.